app/wama.web/Views/Patron/Create.cshtml

```diff
@@ -1,7 +1,7 @@
 ﻿@{
```

Comment:
Yikes! Just realized, this file shouldn't be changed at all. Remember, this is the form for creating a patron account from the check in screen. Hence, it should only work for patron accounts. 

Now it makes more sense, why I thought the comment moved. I was supposed to have this for the _**CreateNewUserAccount.cshtml**_. 

This means, that this file should not be changed at all. Please **revert all changes** on this file.

The changes, I suggested in these comments apply to _**CreateNewUserAccount.cshtml**_